ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/eea/EeaSendRawTransactionTest.java

```diff
@@ -263,7 +261,7 @@ public void validOnChainTransactionPrivacyGroupIsSentToTransactionPool() {
         .thenReturn(optionalPrivacyGroup);
     when(privacyController.buildAndSendAddPayload(
             any(PrivateTransaction.class), any(Bytes32.class), any(String.class)))
-        .thenReturn(Optional.of(ENCLAVE_PUBLIC_KEY));
+        .thenReturn(Optional.of(""));
```

Comment:
That looks a bit funny ... 
Is the test still working with the empty string?

Reply:
removed because it's unused.